README.org

```diff
@@ -60,6 +60,26 @@
                 (aider-minor-mode 1))))
 #+END_SRC
 
+*** Doom integration
```

Comment:
Would be great to put this section above the previous section (Optional), cause there could be lots of users interested in Doom emacs.
Also, if the section include the installation of Doom, maybe change the title to Doom installation and configuration? so that previous installation section can be under a sub section like vanilla emacs, and people can choose installation instruction between previous vanilla one, or doom one.

Reply:
That's great feedback, thank you. I will do that and update the PR